.github/workflows/rust.yml

```diff
@@ -27,31 +27,12 @@ jobs:
             target/
           key: ${{ runner.os }}-cargo-${{ hashFiles('Cargo.lock') }}
       - name: Run Ocular acceptance tests
-        run: cargo test ocular --verbose
-  
-  rust-build:
-    runs-on: ubuntu-latest
-    steps:
-      - name: Checkout branch
-        uses: actions/checkout@v2
-      - name: Set up Rust caches
-        uses: actions/cache@v2
-        id: rust-cache
-        with:
-          path: |
-            ~/.cargo/bin/
-            ~/.cargo/registry/index/
-            ~/.cargo/registry/cache/
-            ~/.cargo/git/db/
-            target/
-          key: ${{ runner.os }}-cargo-${{ hashFiles('Cargo.lock') }}
-      - name: Run Cargo build
-        run: cargo build
-  
+        run: cargo test --verbose
```

Comment:
Looking at this now, `local_single_node_chain_test` doesn't show up too. This means some tests don't get picked up from the ocular library too

Reply:
I see that too. I think what's happening is that the failing `auth_queries` and `bank_queries` tests are preventing it from getting far enough to test those other items. When I skip them locally, it attempts `local_single_node_chain_test` (which fails). I'm going to skip those two tests for now because they fail due to a bad health check that we need to fix.

Reply:
Looks like they work now but the acceptance tests are ignored.